app/responders/ropensci/reviewers_due_date_responder.rb

```diff
@@ -23,7 +23,6 @@ def process_message(message)
       add_to_or_remove_from = [add_or_remove, to_or_from].join(" ")
 
       if ["add to reviewers", "add as reviewer", "assign as reviewer", "assign to reviewers"].include?(add_to_or_remove_from)
-        add_reviewer
```

Comment:
why this?

Reply:
I have no idea how I did that. I was wondering how changing a URL would break tests.